templates/algob-react-boilerplate/assets/fee-check.teal

```diff
@@ -0,0 +1,13 @@
+#pragma version 2
+// Check if the provided fee is at least 10,000 microalgos
+txn Fee
+int 10000
```

Comment:
This value is just for the sake of the template, right?
Also, @ratik21 Can you explain why we are checking for ReKeyTo and CloseRemainderTo?

Reply:
Yes, no. `10k` is just a common value. Lines 3-5 are basically checking if `tx_fee >= 10000`. I think this should be `<=`. This is to ensure that tx fee is not too high, so that we don't burn a lot of algos in a transaction on it's fees.

`RekeyTo` and `CloseRemainderTo` checks are common transaction check (to make sure tx is secure). `CloseRemTo` must be `ZeroAddr` (unless specified otherwise), else someone can empty all your algos to another account. Similarly `rekeyTo` checks that account is secure. 
You can read more about the best practices here - [https://developer.algorand.org/docs/reference/teal/guidelines/](https://developer.algorand.org/docs/reference/teal/guidelines/)